proposals/20200504-falco-artifacts-scope.md

```diff
@@ -0,0 +1,191 @@
+# Falco Scope
+
+As a project we would like to support the following artifacts.
+
+Everything else will be moved to [contrib](https://github.com/falcosecurity/contrib).
+
+As a project we will build, change, rename, and move files, documents, scripts, configurations according to this new state of the world.
+
+Inspired from [#1114](https://github.com/falcosecurity/falco/issues/1114) and many of the weekly community calls.
+
+## Terms
+
+**falco** 
+
+*Standalone Falco binary*
+
+**driver**
+
+*System call provider from the Linux kernel. Either (`bpf`, `module`, or `ptrace`)*
+
+**falco-driver-loader-script**
+
+
+*The bash script found [here](https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader) that tries to compile else download the kernel module.*
+
+**package**
+
+*An installable artifact that is operating system specific. All packages MUST be hosted on bintray for X86_64.*
+
+**image**
+
+*OCI compliant container image hosted on dockerhub with tags for every release.*
+ 
+
+# Packages
+
+Official packages for X86_64 only. All package names should contain a version which is not specified below.
+
+The following convention MUST be used for all packages.
+
+_If a package installs Falco as a daemon (systemd, init) it MUST contain `daemon`._
+
+_If a package installs the Falco kernel module it MUST contain `module`._
+
+_If a package installs the Falco BPF probe it MUST contain `bpf`._
+
+_If a package installs the Falco kernel module it MUST contain `module`._
+
+---
+
+### .deb
+
+ Falco running in debian like systems that will default to the kernel module.
+
+- falco-x86.deb
+     - `falco` only (default depends on `falco-module`)
+ - falco-daemon-x86.deb
+     - `falco` and script to configure as daemon (default depends on `falco-module`)
+ - falco-module-x86.deb
+     - `module` only
+ - falco-bpf-x86.deb
+     - `bpf` only
+ - falco-ptrace-x86.deb
+     - `ptrace` only
+
+### .rpm
+
+ Falco running in rpm like systems that will default to the kernel module.
+
+- falco-x86.deb
+     - `falco` only (default depends on `falco-module`)
+ - falco-daemon-x86.deb
+     - `falco` and script to configure as daemon (default depends on `falco-module`)
+ - falco-module-x86.deb
+     - `module` only
+ - falco-bpf-x86.deb
+     - `bpf` only
+ - falco-ptrace-x86.deb
+     - `ptrace` only
+
+
+### .tar.gz
```

Comment:
I agree with having an INSTALL file into this artifact.

Not really sure about creating 4 artifacts rather than 1. This was not the scope of the discussion about artifacts, IMHO.